core/build.gradle

```diff
@@ -88,7 +88,9 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.13.2'
     testImplementation group: 'com.github.stefanbirkner', name: 'system-rules', version: '1.19.0'
     testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.12.348'
-    testImplementation group: 'org.mock-server', name: 'mockserver-netty', version: '5.15.0'
+    testImplementation group: 'org.mock-server', name: 'mockserver-netty', version: '5.15.0', {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
```

Comment:
Without this the container logs don't work because we have a conflicting sl4j-api version (2.x.y is coming from here, 1.7.36 is coming from :common)